### PR TITLE
Fix content-type header for http2

### PR DESCRIPTION
**Pull request by procoru**

### CODE REVIEW

File: src/http/interceptor/body.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function (request, next) {
 
     if (isFormData(request.body)) {
 
-        request.headers.delete('Content-Type');
+        request.headers.delete('Content-Type') || response.headers.get('content-type');
```
Comment (iflamed) [inline src/http/interceptor/body.js]: Hi, It seems to you change the wrong line.   The line number: 40 should be updated. Could you fix it again.

**Comment by steffans** [comment]
What will this PR do? Since 1.0 `request.headers` methods work case-insensitive.

**Comment by iflamed** [comment]
@steffans It's not working with http2. In my program, the ajax response.data is string when the server response with content-type: application/json, which is a http2 request. After I add the response.headers.get('content-type') at line 40 then working.

**Comment by iflamed** [comment]
@steffans I just check again, The vue-resource which I used is 1.0.2

**Comment by iflamed** [comment]
@steffans  Maybe you should close this pr.I found i used the wrong version of vue-resource.  The package.json is updated when i pull from the laravel/laravel. But i didn't reinstall it. So the 1.0.0+ is working for me.

**Pull request closed without merging**